app/Repositories/BuildRepository.php

```diff
@@ -157,15 +155,34 @@ public function getLastBuild(Application $application, Platform $platform)
      * Return the build grouped by platform
      *
      * @param  Application  $application
+     * @param  int $applicationActiveDevices
      * @return Collection
      */
-    public function getByPlatform(Application $application): Collection
+    public function getByPlatform(Application $application, int $applicationActiveDevices): Collection
     {
         return $application->builds()
             ->latest()
             ->get()
             ->groupBy('platform')
-            ->sortKeys();
+            ->sortKeys()
+            ->map(function (Collection $platformCollection) use ($applicationActiveDevices) {
+                return $platformCollection->map(function (Build $build) use ($applicationActiveDevices) {
+
+                    // clear 'appends', else `installations` and `downloads` will be called by the resource
+                    $build->setAppends([]);
+
+                    if (! $build->partial_rollout && $build->rollout_percentage !== 100) {
```

Comment:
Sorry I don't get this part:

```
&& $build->rollout_percentage !== 100)
```

I mean, why `if !== 100` then we set it to 100? 🤔

Reply:
well, there could be a case when `partial_rollout` was set to `true` with `rollout_percentage == 20%`. Then `partial_rollout` set to 'false', but `rollout_percentage` is still set `20%`.  In this case `rollout_percentage` value should be 100% as no partial rollout is in use.